core/server/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -438,4 +440,57 @@ private void createBlockFile(String blockPath) throws IOException {
     FileUtils.changeLocalFileToFullPermission(blockPath);
     LOG.debug("Created new file block, block path: {}", blockPath);
   }
+
+  /**
+   * This class contains some metrics related to the block worker.
+   */
+  @ThreadSafe
+  public static final class Metrics {
```

Comment:
javadoc this is referenced in `WebInterfaceWorkerMetricsServlet` so that is public